Mathlib/Topology/ContinuousOn.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Sébastien Gouëzel
 
 ! This file was ported from Lean 3 source module topology.continuous_on
-! leanprover-community/mathlib commit bcfa726826abd57587355b4b5b7e78ad6527b7e4
+! leanprover-community/mathlib commit 55d771df074d0dd020139ee1cd4b95521422df9f
```

Comment:
I assume this was ported by hand? I think it would be best to put off reviewing it until the mathport output is available and shown on https://leanprover-community.github.io/mathlib-port-status/file/topology/continuous_on?range=bcfa726826abd57587355b4b5b7e78ad6527b7e4..55d771df074d0dd020139ee1cd4b95521422df9f

Reply:
Yes, I copy+pasted, then fixed compile.